diesel-migrations/2024-08-22-013559_race_events_table/up.sql

```diff
@@ -0,0 +1,18 @@
+CREATE TABLE race_events (
+   id                   INTEGER PRIMARY KEY AUTOINCREMENT NOT NULL,
+   guild_id             TEXT NOT NULL,
+   bracket_race_info_id INTEGER NOT NULL,
+   scheduled_event_id   TEXT NOT NULL UNIQUE,
+
+   FOREIGN KEY(bracket_race_info_id) REFERENCES bracket_race_infos(id) 
+);
+
+CREATE UNIQUE INDEX guild_bri ON race_events(guild_id, bracket_race_info_id);
+
+INSERT INTO race_events (guild_id,             bracket_race_info_id, scheduled_event_id)
+                         -- NOTE: this is the prod server id!
+SELECT                   "982502396590698498", id,                   scheduled_event_id
```

Comment:
i guess a non-gross option would be config in the database instead of a `.env` file. we could establish a configuration table, and either do something like using DB as primary source and env vars as backup, or do something like only have a config table, but have a process that initializes it from env vars at startup

i might think about that later but i'm just gonna deal with this for now